File: tests/fixtures/ws-go/basic/no_imports.go
package main

// File without websocket imports — must produce zero emits.

type fakeUpgrader struct{}

func (u *fakeUpgrader) Upgrade(_, _, _ interface{}) (interface{}, error) {
	return nil, nil
}

func local() {
	u := &fakeUpgrader{}
	u.Upgrade(nil, nil, nil)
}

File: tests/fixtures/ws-rs/basic/src/no_imports.rs
// File without tungstenite use — must produce zero emits.

pub struct FakeBus;

impl FakeBus {
    pub async fn accept_async(&self) {}
    pub async fn connect_async(&self, _url: &str) {}
}

pub async fn local(b: &FakeBus) {
    b.accept_async().await;
    b.connect_async("ws://nope").await;
}
